build/configs/artik053/hello-protected/defconfig

```diff
@@ -832,7 +832,7 @@ CONFIG_LIB_USRWORKSTACKSIZE=2048
 #
 # Binary Loader
 #
-# CONFIG_BINFMT_DISABLE is not set
+CONFIG_BINFMT_ENABLE=y
```

Comment:
Why do we enable binary loader in hello protected?

Reply:
Right. That's why I left "do not merge". I will remove it on protected build when I am in office.